metadata.sdk.in

```diff
@@ -0,0 +1,22 @@
+[Runtime]
+name=io.atom.electron.Sdk
+runtime=io.atom.electron.Platform/@@SDK_ARCH@@/@@SDK_BRANCH@@
+sdk=io.atom.electron.Sdk/@@SDK_ARCH@@/@@SDK_BRANCH@@
+
+[Extension org.freedesktop.Platform.GL]
+version=@@SDK_RUNTIME_VERSION@@
+directory=lib/GL
+
+[Extension org.freedesktop.Platform.Timezones]
+version=@@SDK_RUNTIME_VERSION@@
+directory=share/zoneinfo
+
+[Extension org.freedesktop.Platform.GStreamer]
```

Comment:
Should I remove this? Copied it in from the gnome runtime, but I don't think chromium uses gstreamer...